client/me/account/main.jsx

```diff
@@ -535,12 +536,12 @@ const Account = React.createClass( {
 				{ this.communityTranslator() }
 
 				{ config.isEnabled( 'me/account/color-scheme-picker' ) &&
+					supportsCssCustomProperties() &&
```

Comment:
I think we could shuffle a few of the changes around to better isolate the changesets if you don't mind?
We could remove the implementation from this PR and add that elsewhere, allowing us to merge the addition of the `feature-detection` lib before continuing with #17259.
Because we'd then have `feature-detection` available we could add the check for `supportsCssCustomProperties` right there.

Reply:
Commenting for continuity: As per our discussion in Slack just now, we're going to leave the PR order as is to ensure we have a good test case for the feature detection. We're aiming to merge the PRs in quick succession.